### DISCUSSION TITLE
Disable "keep me signed in"

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1234] [Q&A] [answered]
I want auto log out  when idle for too long of a period...How can I set that and disable the "Keep me signed in" checkbox.

**Comment by meetmandeep** [comment, marked as answer]
Edit your web.config and update the PersistentCookieTimeout value (In Minutes)

`<add key="PersistentCookieTimeout" value="20160" />`

You can disable the "Keep me signed in" checkbox from Sign In Form Block Settings:

![image](https://user-images.githubusercontent.com/29958123/118054034-6b069280-b33a-11eb-91d4-87cada7f7478.png)

![image](https://user-images.githubusercontent.com/29958123/118053972-53c7a500-b33a-11eb-8802-804a0e99e964.png)

**Reply by scottw12** [reply]
Thanks
